### PR TITLE
Change default ALT PIDs for fixed wing

### PR DESCRIPTION
**Pull request by shellixyz**

### CODE REVIEW

**Comment by giacomo892** [comment]
@shellixyz Perhaps better to change the comments too?

File: src/main/flight/pid.c [hunk 1]
```diff
@@ -156,9 +156,9 @@ PG_RESET_TEMPLATE(pidProfile_t, pidProfile,
                 },
                 [PID_HEADING] = { 60, 0, 0 },
                 [PID_POS_Z] = {
-                    .P = 50,    // FW_POS_Z_P * 100
-                    .I = 0,     // not used
-                    .D = 0,     // not used
+                    .P = 40,    // FW_POS_Z_P * 100
```
Comment (digitalentity) [inline src/main/flight/pid.c]: @shellixyz since you are touching this part of code - please also fix comments :)

**Review by digitalentity — DISMISSED** [review]

File: src/main/flight/pid.c (hunk 1 above)
Comment (shellixyz) [reply src/main/flight/pid.c]: Indeed good idea. Done.

**Comment by fiam** [comment]
Resubmitted as #3597, to workaround the problem with Travis

**Pull request closed without merging**